### PR TITLE
feat: add lensIso

### PR DESCRIPTION
**Pull request by char0n**

### CODE REVIEW

**Comment by char0n** [comment]
Ref: #178

**Comment by char0n** [comment]
@BjornMelgaard, @rockymadden could you do a code review pls ? Thank you

**Comment by BjornMelgaard** [comment]
Interesting function, all is fine, something like this will work

```
        interface Isomorpism {
            (x: any): any
            from(y: any): any
        }

        /**
         * Defines an isomorphism that will work like a lens. It takes two functions.
         * The function that converts and the function that recovers.
         */
        lensIso(to: Function, from: Function): Isomorpism
        lensIso(to: Function): (from: Function) => Isomorpism
```
BTW: `Function` == `(...x: any[]): any`

**Comment by char0n** [comment]
You misunderstood the the `from` is registered.

`lensIso.from` - from exists as an attribute of lensIso function, not as an attribute of it's result.

**Comment by BjornMelgaard** [comment]
```
        lensIso: {
            (to: Function, from: Function) => Function
            (to: Function): (from: Function) => Function
            from(lens: Function): Function
        }
```

**Comment by char0n** [comment]
@BjornMelgaard thx I'll do that

File: src/lensIso.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.19.0|1.19.0}
+ * @category Relation
+ * @sig
+ *   (s -> a) -> (a -> s) -> Lens s a
+ *   Lens s a = Functor f => (a -> f a) -> s -> f s
+ * @param {!function} to The function that converts
+ * @param {!function} from The function that recovers
+ * @return {!function} The isomorphic lens
+ *
```
Comment (char0n) [inline src/lensIso.js]: drop empty line

File: src/lensIso.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ * The function that converts and the function that recovers.
+ *
+ * @func lensIso
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.19.0|1.19.0}
+ * @category Relation
+ * @sig
+ *   (s -> a) -> (a -> s) -> Lens s a
+ *   Lens s a = Functor f => (a -> f a) -> s -> f s
+ * @param {!function} to The function that converts
```
Comment (char0n) [inline src/lensIso.js]: Add @see for http://ramdajs.com/docs/#lens

**Pull request merged**